docker-compose.yml

```
@@ -3,51 +3,187 @@
+      - OTEL_RESOURCE_HOST_NAME=${OTEL_RESOURCE_HOST_NAME}
+      - SIGNOZ_ENDPOINT=${SIGNOZ_ENDPOINT}
+      - SIGNOZ_INGESTION_KEY=${SIGNOZ_INGESTION_KEY}
+
+
+  product-service-1:
+    container_name: nsh-store-1
+    build:
+      context: .
+      dockerfile: ./product-service/Dockerfile
+    env_file:
+       - .env
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)